nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,38 @@
+const express = require("express");
+const app = express();
+
+const mealsRouter = require("./routes/meals.js");
+const mealRouter = require("./routes/meal.js");
+const cheapMealsRouter = require("./routes/cheap-meals.js");
+const largeMealsRouter = require("./routes/large-meals.js");
+const reservationsRouter = require("./routes/reservations.js");
+const reservationRouter = require("./routes/reservation.js");
+
+app.get("/", async(request, response) => {
+    response.send("Meal Sharing web application");
+});
+app.get("/meals", async(request, response) => {
+    response.send(mealsRouter);
+});
+
+app.get("/meal", (request, response) => {
+    response.send(mealRouter);
+});
+
+app.get("/cheap-meals", (request, response) => {
+    response.send(cheapMealsRouter);
+});
+
+app.get("/large-meals", (request, response) => {
+    response.send(largeMealsRouter);
+});
+
+app.get("/reservations", (request, response) => {
+    response.send(reservationsRouter);
+});
+
+app.get("/reservation", (request, response) => {
+    response.send(reservationRouter);
+});
```

Comment:
These routes won't work. You're using `Router` in your route files (read about it [here](https://expressjs.com/en/guide/routing.html#express-router) and [here](https://stackoverflow.com/a/33261362)). A `Router` object is like a mini app in a separate file, serving only one or a few specific route(s) that you created it for. Your main app is `app` in app.js. Therefore you don't want to create new routes in app.js, instead you want it to _use_ the routers (mini apps) you created in separate files. 

So to fix this, replace each of these `app.get()` with `app.use()`, passing it the route and the router object you imported, like:

```
/*
// Replace this
app.get("/meals", async(request, response) => {
    response.send(mealsRouter);
});
*/

// with this:
app.use("/meals", mealsRouter ); // notice app.USE ❕ 
```